client/components/status/status.js

```
@@ -926,14 +984,19 @@ export default class Status extends React.Component {
   };
 
   // eslint-disable-next-line class-methods-use-this
-  getDateTimeFormat = (language, time_option, date) => {
+  getDateTimeFormat(language, time_option, date) {
+    if (!date) return "-";
+
+    const d = new Date(date);
+
+    if (isNaN(d.getTime())) return "-";
```

ESLint

[error] Unexpected use of 'isNaN'. Use Number.isNaN instead https://github.com/airbnb/javascript#standard-library--isnan

(no-restricted-globals)